internal/ingress/annotations/annotations.go

```diff
@@ -205,3 +206,12 @@ func ExtractMethods(anns map[string]string) []string {
 	}
 	return strings.Split(val, ",")
 }
+
+// ExtractSNI extracts the route SNI match criteria annotation value.
```

Comment:
docstring doesn't match function name

```suggestion
// ExtractSNIs extracts the route SNI match criteria annotation value.
```